apps/E2E/src/Spinner/specs/Spinner.spec.ios.ts

```diff
@@ -0,0 +1,22 @@
+// import NavigateAppPage from '../../common/NavigateAppPage';
+// import ActivityIndicatorPageObject from '../pages/ActivityIndicatorPageObject';
+// import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT } from '../../common/consts';
+
+// // Before testing begins, allow up to 60 seconds for app to open
+// describe('Activity Indicator Testing Initialization', function () {
+//   it('Wait for app load', async () => {
+//     await NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+//     await expect(await NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+//   });
+
+//   it('Click and navigate to Activity Indicator test page', async () => {
+//     await ActivityIndicatorPageObject.mobileScrollToComponentButton();
+//     await ActivityIndicatorPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+//     /* Click on component button to navigate to test page */
+//     await NavigateAppPage.clickAndGoToActivityIndicatorPage();
+//     await ActivityIndicatorPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
```

Comment:
Are these moves from existing activity indicator test page - https://github.com/microsoft/fluentui-react-native/blob/main/apps/E2E/src/ActivityIndicator/specs/ActivityIndicator.spec.ios.ts. Do you plan to enable these back for Spinner as part of the restructuring? If not in this PR - maybe just remove all this commented out code/ mention a tracking issue to do it later?

Reply:
I have removed for other platform as this PR targets for Android. I have added setup for E2E test for Android. 
E2E tests can added later.